prelude/README.md

```diff
@@ -111,6 +122,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+    (+) 1 ((*) 2 (const 1 2)) = (+) 1 $ (*) 2 $ const 1 2
```

Comment:
I was trying to make sure the examples ran with only the Prelude. Is it kosher to do otherwise?